x-pack/plugins/security_solution/public/common/components/visualization_actions/use_add_to_existing_case.test.tsx

```diff
@@ -62,11 +64,21 @@ describe('useAddToExistingCase', () => {
     expect(result.current.disabled).toEqual(false);
   });
 
-  it("button disabled if user Can't Crud", () => {
-    (useGetUserCasesPermissions as jest.Mock).mockReturnValue({
-      crud: false,
-      read: true,
-    });
+  it("disables the button if the user can't create but can read", () => {
+    (useGetUserCasesPermissions as jest.Mock).mockReturnValue(readCasesPermissions());
+
+    const { result } = renderHook(() =>
+      useAddToExistingCase({
+        lensAttributes: kpiHostMetricLensAttributes,
+        timeRange,
+        onAddToCaseClicked: mockOnAddToCaseClicked,
+      })
+    );
+    expect(result.current.disabled).toEqual(true);
+  });
+
+  it("disables the button if the user can't read but can create", () => {
```

Comment:
Just out of curiosity, is there a scenario where this would ever actually happen?

Reply:
Not currently. You can't customize the roles in such a way to have create but not read.